dev/com.ibm.ws.microprofile.config_fat/test-applications/simultaneousRequests.war/src/com/ibm/ws/microprofile/appConfig/simultaneousRequests/test/SimultaneousRequestsTestServlet.java

```diff
@@ -10,75 +10,19 @@
  *******************************************************************************/
 package com.ibm.ws.microprofile.appConfig.simultaneousRequests.test;
 
-import java.io.IOException;
-import java.io.PrintWriter;
-import java.util.concurrent.atomic.AtomicBoolean;
-
 import javax.inject.Inject;
-
 import javax.servlet.annotation.WebServlet;
-import javax.servlet.ServletException;
-import javax.servlet.ServletOutputStream;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import org.eclipse.microprofile.config.Config;
-import org.eclipse.microprofile.config.spi.ConfigBuilder;
-import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
-import org.junit.Test;
-
-import com.ibm.ws.microprofile.appConfig.test.utils.TestUtils;
 
 import componenttest.app.FATServlet;
 
 @SuppressWarnings("serial")
 @WebServlet("/")
-public class SimultaneousRequestsTestServlet extends HttpServlet {
-
-    @Inject 
-    Config config;
+public class SimultaneousRequestsTestServlet extends FATServlet {
 
-    private static AtomicBoolean firstRequest = new AtomicBoolean(true);
+    @Inject
+    TestBean testBean;
 
-    @Override
-    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        TestUtils.assertContains(config, "onlyInFile", "onlyInFile.fileValue"); //Ensure that a Config object is actually associated with both requests before either destroys it's Config. 
-    
-        Exception e = null;
-        if (firstRequest.compareAndSet(true, false)) {
-            try {
-                delayFirstThread();
-                TestUtils.assertContains(config, "onlyInFile", "onlyInFile.fileValue");
-            } catch (Exception ex) {
-                e = ex;
-            }
-        } else {
-            TestUtils.assertContains(config, "onlyInFile", "onlyInFile.fileValue");
-            try { 
-                synchronized (firstRequest) {
-                    firstRequest.notifyAll();
-                }
-            } catch (Exception ex) {
-                e = ex;
-            }
-        }
-
-        PrintWriter out = response.getWriter();
-
-        if (e == null) {
-            out.println("\nNo exceptions were thrown. Config object: " + config.toString());
-        } else {
-            out.println("\nAn exception was thrown: " + e.toString() + " Config object: " + config.toString());
-        }
-        
+    public void testSimultaneousRequests() throws InterruptedException {
+        testBean.testSimultaneousRequests();
```

Comment:
What's the point of moving this test into another class?

Reply:
oh that's me being over paranoid ... it isolates the cdi function from the webcontainer